docs/pages/guides/cache-builds-remotely.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Use remote build cache providers
+sidebar_title: Cache builds remotely
+description: Accelerate local development by caching and reusing builds from a remote provider.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+Remote build cache is an experimental feature that allow users to search for a build with a matching fingerprint on a remote server when running `npx expo run:[ios|android]`.
+If a cached build exists, it downloads and launches it rather than compiling locally. Otherwise, it proceeds with a local build as usual and then uploads the resulting binary to the remote cache for future runs.
+
+## Using EAS as a remote build provider
+
+To use the EAS remote build provider plugin start by installing the `eas-build-cache-provider` package as a developer dependency:
+
+<Terminal cmd={['$ npx expo install eas-build-cache-provider']} />
+
+Then update your **app.json** to
+
+```json app.json
+{
+  "expo": {
+    ...
+    "experiments": {
+      "remoteBuildCache": {
+        "provider": "eas-build-cache-provider"
+      }
+    }
+  }
+}
+```
+
+## Creating a custom remote build provider
+
```

Comment:
let's add a link here to the github cache provider in the examples directory

Reply:
added